packages/google_adsense/example/integration_test/experimental_ad_unit_widget_test.dart

```diff
@@ -19,6 +19,17 @@ import 'js_interop_mocks/adsense_test_js_interop.dart';
 const String testClient = 'test_client';
 const String testSlot = 'test_slot';
 
+class CallbackTracker {
+  void Function() createCallback() {
+    final int callbackIndex = _callbackStates.length;
+    _callbackStates.add(false);
+    return () => _callbackStates[callbackIndex] = true;
+  }
+
+  bool get allCalled => _callbackStates.every((bool state) => state);
+  final List<bool> _callbackStates = <bool>[];
```

Comment:
(Note to self: This does not need a "reset" method or similar, because each test creates its own instance of the `CallbackTracker`)